web/static/_redirects

```diff
@@ -8,6 +8,7 @@
 /docs/deploying                    /docs/deployment/intro                    301
 /docs/advanced/deployment/overview /docs/deployment/intro                    301
 /docs/integrations/css-frameworks  /docs/project/css-frameworks             301
+/docs/data-model/backends          /docs/data-model/databases               301
```

Comment:
Hm but this is probably not 1 on 1 mapping? I am guessing that backends was wider than just databases? If it is just name change then makews sense, but if not and backends was wider, maybe better not to map at all?